### PR TITLE
feat(#6): add commitlint

### PR DESCRIPTION
**Pull request by xiaoyuweiliang**

### CODE REVIEW

File: .husky/commit-msg
Comment (Wenrh2004) [inline .husky/commit-msg:1]: Only just?
You need add pre-commit configuration.

File: package.json
Comment (Wenrh2004) [inline package.json:1]: Why add it the eslint-config-next

**Review by Wenrh2004 — CHANGES_REQUESTED** [review]
Need fix

File: package.json
Comment (PolarishT) [reply package.json:1]: +1

**Review by PolarishT — CHANGES_REQUESTED** [review]
改吧

**Comment by xiaoyuweiliang** [comment]
feat: add commirlint

**Pull request closed without merging**